modular_skyrat/modules/modular_items/lewd_items/code/lewd_machinery/lustwish.dm

```diff
@@ -12,107 +12,148 @@
-					/obj/item/clothing/ears/kinky_headphones = 5,
-					/obj/item/clothing/mask/gas/bdsm_mask = 5,
-					/obj/item/reagent_containers/cup/lewd_filter = 5,
-					/obj/item/clothing/glasses/hypno = 4,
-					/obj/item/clothing/head/costume/kitty = 4,
-					/obj/item/clothing/head/costume/rabbitears = 4,
-
-
-					//neck
-					/obj/item/clothing/neck/kink_collar = 8,
-					/obj/item/clothing/neck/human_petcollar = 8,
-					/obj/item/clothing/neck/human_petcollar/choker = 8,
-					/obj/item/clothing/neck/human_petcollar/thinchoker = 8,
-					/obj/item/clothing/neck/human_petcollar/locked/cow = 8,
-					/obj/item/clothing/neck/human_petcollar/locked/bell = 8,
-					/obj/item/clothing/neck/human_petcollar/locked/cross = 8,
-					/obj/item/clothing/neck/human_petcollar/locked/spike = 8,
-					/obj/item/clothing/neck/size_collar = 8,
-
-					//torso clothing
-					/obj/item/clothing/under/misc/latex_catsuit = 8,
-					/obj/item/clothing/suit/straight_jacket/latex_straight_jacket = 5,
-					/obj/item/clothing/under/costume/maid = 5,
-					/obj/item/clothing/under/rank/civilian/janitor/maid = 5,
-					/obj/item/clothing/under/costume/lewdmaid = 5,
-					/obj/item/clothing/suit/straight_jacket/shackles = 4,
-					/obj/item/clothing/under/stripper_outfit = 5,
-					/obj/item/clothing/under/costume/bunnylewd = 5,
-					/obj/item/clothing/under/costume/bunnylewd/white = 5,
-					/obj/item/clothing/under/misc/skyrat/gear_harness = 4,
-
-					//hands
-					/obj/item/clothing/gloves/ball_mittens = 8,
-					/obj/item/clothing/gloves/latex_gloves = 8,
-					/obj/item/clothing/gloves/evening = 5,
-
-					//legs
-					/obj/item/clothing/shoes/latex_socks = 8,
-					/obj/item/clothing/shoes/latex_heels = 4,
-					/obj/item/clothing/shoes/latex_heels/domina_heels = 4,
-					/obj/item/clothing/shoes/jackboots/knee = 3,
-					/obj/item/clothing/under/pants/skyrat/chaps = 4,
-
-					//belt
-					/obj/item/clothing/strapon = 6,
-					/obj/item/storage/belt/erpbelt = 5,
-
-					//chems
-					/obj/item/reagent_containers/pill/crocin = 20,
-					/obj/item/reagent_containers/pill/camphor = 10,
-					/obj/item/reagent_containers/cup/bottle/crocin = 6,
-					/obj/item/reagent_containers/cup/bottle/camphor = 3,
-					/obj/item/reagent_containers/cup/bottle/succubus_milk = 6, //Those are legal 'cause you can just turn off prefs in round in "CLOWN SMOKE MACHINE+PENIS ENLARGEMENT CHEMICAL CASE". Yes, i have special code-phrase for this. I've seen some shit.
-					/obj/item/reagent_containers/cup/bottle/incubus_draft = 6,
-
-					//special
-					/obj/item/clothing/glasses/nice_goggles = 1, //easter egg, don't touch plz)
-
-					//fur niture //haha you got it
-					/obj/item/storage/box/bdsmbed_kit = 4,
-					/obj/item/storage/box/strippole_kit = 4,
-					/obj/item/storage/box/xstand_kit = 4,
-					/obj/item/storage/box/milking_kit = 4,
-					/obj/item/storage/box/shibari_stand = 4)
+	product_categories = list(
+		list(
+			"name" = "Toy",
+			"icon" = FA_ICON_MICROPHONE,
```

Comment:
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/131064729/a4d18df6-cc51-462f-9e54-e0bb0efbded3)
for reference. Fun fact? Vendors don't like it when you have the same item in multiple categories. There IS a testing category that uses the Grin icon but since the code forces the items in it to be in other categories instead, that leaves Grin with 0 items in it and thus it doesn't show. Doesn't matter here, because Wand is very clearly the correct choice.

Reply:
I'd probably make the categories plurals. Toy implies it's just one and such.
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/102828457/02fc0e0b-ab06-432e-9597-b0c93c3c8c42)
Example.